birdbox/.env-example

```diff
@@ -1,11 +1,21 @@
 DEBUG=False
 DISABLE_SSL=True
+RATELIMIT_ENABLE=False
 
-# Change to True if you want to use SSO locally, else you'll use username+password auth
-USE_SSO_AUTH=False
+EMAIL_BACKEND="django.core.mail.backends.console.EmailBackend"
+
+# If you want to use Google Cloud Storage locally, you'll need to set these
+# variables. See https://django-storages.readthedocs.io/en/latest/backends/gcloud.html
+GS_BUCKET_NAME="gcs-bucket-name-here"
```

Comment:
What are we using GCS buckets for? And shouldn't these be commented out if this is meant to be coppied for an initial working `.env`?

Reply:
Uploading images for pages I assume? I just don't see anything else in this PR dealing with that.

Reply:
GCS buckets are where we store editor-uploaded image for Wagtail.

And yeap, this is a template for a local .env file (with the assumption that one might want to test/work on something with cloud storage)

Reply:
And yeah, this was a case of mopping up as I was going - not ideal to bundle into this branch, I admit